docs/Externals.md

```diff
@@ -23,6 +23,45 @@ For other calls, such as the Address API, it is essentially impossible to do wit
 This is because the Address API relies on a unique security token from Amazon, 
 and the only way to get it is to be running inside Alexa. An emulator like Virtual Alexa just cannot do it.
 
+## DynamoDB
+To mock calls to DynamoDB, simply enable our DynamoDB mock service.
+
+Example:
+
+```javascript
+const virtualAlexa = VirtualAlexa.Builder()
+    .handler("index.handler")
+    .interactionModelFile("models/en-US.json")
+    .create();
+
+virtualAlexa.dynamo().mock();
+```
+
+The Dynamo DB mock will be called automatically for get and put requests to Dynamo, without hitting the service itself.
+
+Records will be stored locally in memory instead of in Dynamo.
+
+Dynamo can be used in the tests by simply calling the Dynamo SDK in the normal way:
+
+```
+const getParams = {
+    Key: {
+        ID: {
+            S: "The Beatles",
+        },
+    },
+    TableName: "Musicians",
+};
+const dynamo = new AWS.DynamoDB();
+dynamo.getItem(getParams, function(error, data) {
+    assert.equal(data.Item.Genre.S, "Rock");
+});
+```
+
+Complete information on [AWS SDK for Node.js is here](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html).
```

Comment:
this links to dynamo documentation which is the relevant part for this, but kinda refers to the whole AWS SDK on the name, maybe [DynamoDB for Node.js is here]?